packages/client/src/v2-events/messages/advancedSearch.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+import { defineMessages, MessageDescriptor } from 'react-intl'
+
+interface IAdvancedSearchMessages
+  extends Record<string | number | symbol, MessageDescriptor> {
+  birthTabTitle: MessageDescriptor
+  birthTabTitleExport: MessageDescriptor
+  deathTabTitle: MessageDescriptor
+  deathTabTitleExport: MessageDescriptor
+  advancedSearch: MessageDescriptor
+  advancedSearchInstruction: MessageDescriptor
+}
+
+const messagesToDefine: IAdvancedSearchMessages = {
```

Comment:
Place these message items directly to the component files where the message is used. So let's steer away from these "messages" files and instead just do 

```ts
const messages = defineMessages({...})
```
in the view module's root